app/jobs/update_civic.rb

```diff
@@ -2,4 +2,23 @@ class UpdateCivic < ApiUpdater
   def updater
     return Genome::OnlineUpdaters::Civic::Updater.new()
   end
+
+  def next_update_time
+    Date.today
+      .beginning_of_week
+      .next_day(14)
+      .midnight
+  end
+
+  def should_group_genes?
+    return true
```

Comment:
This will obviously work just fine and there's no need to change it, but idiomatically, `return` should really only be used for flow control in ruby. The method will return the return value of the last line anyways.